docs/design/coreclr/botr/clr-abi.md

```diff
@@ -322,33 +322,11 @@ Finally1:
 
 Note that JIT64 does not implement this properly. The C# compiler used to always insert all necessary "step" blocks. The Roslyn C# compiler at one point did not, but then was changed to once again insert them.
 
-## The PSPSym and funclet parameters
+## Funclet parameters
 
-The *PSPSym* (which stands for Previous Stack Pointer Symbol) is a pointer-sized local variable used to access locals from the main function body.
+For filter funclets the VM sets the frame register to be the same as the parent function. For second pass funclets the VM restores only the frame register on AMD64 and all non-volatile registers on all other platforms.
```

Comment:
The deleted "NativeAOT" comment says that 2nd pass funclets are the same on AMD64 and other platforms. Should you keep this comment?

Is there a reason why AMD64 only restores the frame pointer and not all the non-volatile registers for non-NativeAOT?

Reply:
I originally planned to align AMD64 and restore the registers too, but I reverted it in https://github.com/dotnet/runtime/pull/114630/commits/c97c7cf6f0aafdc0e8cf454a948c3ff5623778a7. There was a concern that the additional state saving/restoring may affect performance. Given that JIT doesn't take advantage of the restored context (https://github.com/dotnet/runtime/pull/114630#issuecomment-2810210759) I decided to revert to the previous CoreCLR behavior. We can address this separately.

Reply:
(Fixing the documentation should be done in this PR though, so I will try to rewrite it to keep the original NativeAOT comment.)

Reply:
There was duplicate information between `Funclet parameters` and `Registers on entry to a funclet` sections. I moved all the information about registers to the later.